ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-main.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_builder_rhel-8-base-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-base-openshift-4.8
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: ocp_builder_rhel-8-base-openshift-4.8
+  inputs:
+    ocp_builder_rhel-8-base-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.8
+  to: ironic-agent
+promotion:
+  name: "4.8"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
+- as: e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
+- as: e2e-metal-ipi-virtualmedia
```

Comment:
does this apply to virtualmedia scenarios?

Reply:
We will switch everything to the new container.